CDCgov/prime-reportstream
# 7628 - Pagination usage with feature info and membership data

This PR ...

Captures the number of times any given page number is viewed on each table.

Test Steps:
1. Log into RS
2. Navigate to either Daily Data or Submission History
3. Select the next page number or the Next arrow button
App Insights should capture the event name ("Daily Data | Pagination Filter", "Submission History | Pagination Filter") along with the page number that was clicked and the page size within the Logs or Usage Events

## Changes
- Added onPaginationClick to UsePagination.ts to track pagination events from SubmissionTable.tsx and DeliveriesTable.tsx.

## Checklist

### Testing
